### ISSUE TITLE
&nbsp; error on update of a richtext property with a mapping

### ISSUE THREAD

**Issue by gfoo** [dhlab-basel/Knora#1071]
Get this error when a `&nbsp;` is detected in the xml value:
``` 
{"error":"org.knora.webapi.StandoffConversionException: The provided XML could not be parsed: org.xml.sax.SAXParseException; lineNumber: 2; columnNumber: 16; The entity \"nbsp\" was referenced, but not declared."}
``` 
 To reproduce, just add a `&nbsp;`  in the xml value:
``` 
"knora-api:textValueAsXml" : "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<text><p>&nbsp;value</p></text>",
``` 
Seems logical to me (it works with `&#160;`), but maybe any special rules (as mentioned by @benjamingeer in #1069) could be added for these well known html entities.

**Comment by benjamingeer** [comment]
@tobiasschweizer What do you think about this?

**Comment by tobiasschweizer** [comment]
I always use `&#160;`. `&nbsp;` would have to be declared since it is not a predefined entity (https://en.wikipedia.org/wiki/List_of_XML_and_HTML_character_entity_references#Predefined_entities_in_XML).

I believe that @loicjaouen solved that problem in CKeditor once.

The root of the problem is that Knora expects XML while CKeditor produces HTML in strict XML notation.

When we implemented standoff for Knora, we got rid of the old standoff API and switched to XML. Then we had to make that work with CKeditor.
